source/contribute.rst

```diff
@@ -57,31 +57,42 @@ Initial Setup
     $ cd orm
     $ git remote add doctrine git://github.com/doctrine/orm.git
 
--  Adjust your branch to track the doctrine master remote branch, by
-   default it'll track your origin remote's master:
+Branching from the default branch
+---------------------------------
 
-.. code-block:: console
+New pull requests are created with the repository's default branch as base branch.
+The default branch is the branch you see when you enter the repository page on GitHub.
+
+.. image:: ../images/default-branch.png
+   :alt: The default branch
+   :style: margin-bottom: 20px
 
-    $ git config branch.master.remote doctrine
+In this ORM example, it's the branch with the name **2.7**. The branch name reflects the
+current lowest supported version of a repository.
 
-Keeping your master up-to-date!
--------------------------------
+Newly introduced changes to 2.7 will be up-merged at a later point in time
+to newer version branches (e.g. 2.8, 3.0). This way you don't have to
+re-introduce a new fix or feature of 2.7 with another pull request to
+the other version branches.
 
-Once all this is done, you'll be able to keep your local master up to
+Keeping the default branch up-to-date!
```

Comment:
Maybe we should remove that paragraph entirely, or reduce it to `git fetch`. IMO, people should use `git switch --create my-branch origin/2.8.x`, not `git switch --create my-branch 2.8.x`

Reply:
I've reduced it to `git fetch`.